src/main/resources/Service.csv

```diff
@@ -0,0 +1,2 @@
+service_code,ccd_service_name,ccd_jurisdiction_name
+AA1,"service1,serive2",test-jurisdiction
```

Comment:
file is wrong

there are 3 columns in file. - 
1.  SERVICE ID 
2.  CCD Jurisdiction Name 
3. CCD CASE TYPE

the third column which is CCD case type is a comma separated value and not the second one